src/reducers/network/system/model.js

```diff
@@ -30,13 +30,13 @@ const globalDebtCeiling = addresses => ({
 const debtAuctionLotSzie = addresses => ({
   target: addresses.MCD_VOW,
   call: ['sump()(uint256)'],
-  returns: [[DEBT_AUCTION_LOT_SIZE, val => DAI(val, -18)]]
+  returns: [[DEBT_AUCTION_LOT_SIZE, val => DAI(val, -45)]]
```

Comment:
you can do `DAI.rad` here

Reply:
Thanks, updated.  There's some other parts of the portal code that can be reworked to use the Currency math as well